packages/camera/tizen/src/camera_device.cc

```diff
@@ -343,41 +343,26 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
           [this](size_t width,
                  size_t height) -> const FlutterDesktopGpuBuffer * {
             std::lock_guard<std::mutex> lock(mutex_);
-            if (prepared_packet_ && !IsValidMediaPacket(prepared_packet_)) {
-              media_packet_destroy(prepared_packet_);
-              prepared_packet_ = nullptr;
-            }
-            if (current_packet_ && !IsValidMediaPacket(current_packet_)) {
```

Comment:
Could you remove `IsValidMediaPacket` definition too.

Reply:
@bbrto21 Oh. good point!